core/core_modules/admin.js

```diff
@@ -0,0 +1,170 @@
+﻿var config = require.once('../config.js'),
+
+getHasPermission = function (config, userId, name, threadId, moduleName) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 space characters but found 0.](https://www.codacy.com/app/theprojectshop/Kassy/file/2556616938/issues/source?bid=3282987&fileBranchId=3292072#l3)